CAM-11283 - Custom Event Timeout (#2804)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Core Concepts/lifecycle-editor.md

{{{
"title": "Lifecycle Editor",

> "date": "09-01-2016",

Reviewer: Please, update date

Author: ACK

![Save the changes when close Editor](../../images/cloud-application-manager/lifecycle-editor-3-1.png)

**Edit Event Timeout**

> You can set up a maximum execution timeout in an event script in order to avoid long-consuming executions. Execution timeout can be defined in [Script boxes](../Automating Deployments/script-box.md) and at instance level.

Reviewer: ```suggestion
You can set up a maximum execution timeout in an event script of the instance in order to control long-lasting executions. Execution timeout can also be defined in [Script boxes](../Automating Deployments/script-box.md).
```

Author: ACK

> If script has an execution timeout, clock icon will be shown after event-name.

Reviewer: ```suggestion
If a script has an execution timeout, a clock icon will be shown next to the event name.
```

Author: ACK

> If a script has an execution timeout,a clock icon will be shown next to the event name.

Reviewer: ```suggestion
If a script has an execution timeout, a clock icon will be shown next to the event name.
```

Author: Done

![Event with defined timeout in lifecycle editor](../../images/cloud-application-manager/lifecycle-editor/event-timeout-1.png)

> Action menu allows to modify setted timeout at instance level, without affect box-level configuration.

Reviewer: ```suggestion
The action menu allows you to set or modify the timeout value for the instance, without affecting the source box configuration.
```

Author: ACK

> The action menu allows you to set or modify the timeout value for the instance, without affecting the source box configuration.

Reviewer: ```suggestion
The action menu of an event allows you to set or modify the timeout value for the instance script event, without affecting the source box configuration.
```

Author: ACK

![Save the changes when close Editor](../../images/cloud-application-manager/lifecycle-editor-3-1.png)

**Edit Event Timeout**

> Events allow operator to setup a maximum execution timeout in order to avoid long-consuming executions. Execution timeout can be defined in [Script boxes](../Automating Deployments/script-box.md) and modified at instance level.

Reviewer: ```suggestion
You can set up a maximum execution timeout in an event script in order to avoid long-consuming executions. Execution timeout can be defined in [Script boxes](../Automating Deployments/script-box.md) and modified at instance level.
```
Is it possible to just set it at the instance level, even if it has not been defined at box level? I guess so, you should update wording to reflect it

If script has an execution timeout, clock icon will be shown after event-name.

> ![event-timeout-1.png](../../images/cloud-application-manager/lifecycle-editor/event-timeout-1.png)

Reviewer: Please, provide meaningful alt texts (review all images alt texts)
